lib/tasks/javascript.rake

```diff
@@ -0,0 +1,100 @@
+def public_js
+  "#{Rails.root}/public/javascripts"
+end
+
+def vendor_js
+  "#{Rails.root}/vendor/assets/javascripts"
+end
+
+def library_src
+  "#{Rails.root}/node_modules"
+end
+
+task 'javascript:update' do
+
+  require 'uglifier'
+
+  yarn = system("yarn install")
+  abort('Unable to run "yarn install"') unless yarn
+
+  dependencies = [
+    {
+      source: 'ace-builds/src-min',
+      destination: 'ace',
+      public: true
+    }, {
+      source: 'chart.js/dist/Chart.min.js',
+      public: true
+    }, {
+      source: 'magnific-popup/dist/jquery.magnific-popup.min.js',
+      public: true
+    }, {
+      source: 'pikaday/pikaday.js',
+      public: true
+    }, {
+      source: 'spectrum-colorpicker/spectrum.js',
+      uglify: true,
+      public: true
+    }, {
+      source: 'spectrum-colorpicker/spectrum.css',
+      public: true
+    }, {
+      source: 'favcount/favcount.js'
+    }, {
+      source: 'handlebars/dist/handlebars.js'
+    }, {
+      source: 'handlebars/dist/handlebars.runtime.js'
+    }, {
+      source: 'htmlparser/lib/htmlparser.js'
+    }, {
+      source: 'jquery-resize/jquery.ba-resize.js'
+    }, {
+      source: 'jquery.autoellipsis/src/jquery.autoellipsis.js',
+      destination: 'jquery.autoellipsis-1.0.10.js'
+    }, {
+      source: 'jquery-color/jquery.color.js'
+    }, {
+      source: 'jquery.cookie/jquery.cookie.js'
+    }, {
+      source: 'jQuery/dist/jquery.js'
+    }, {
+      source: 'jquery-tags-input/src/jquery.tagsinput.js'
+    }, {
+      source: 'markdown-it/dist/markdown-it.js'
+    }, {
+      source: 'mousetrap/mousetrap.js'
```

Comment:
@pmusaraj Let's skip mousetrap until https://github.com/discourse/discourse/pull/6436/files#r221823934 is resolved